feature-group-definitions/tabindex.yml

```
@@ -1,4 +1,5 @@
 name: tabindex
+description: The `tabindex` HTML attribute makes elements focusable, and defines the elements' relative ordering for sequential focus navigation.
```

```suggestion
description: The `tabindex` HTML attribute makes elements focusable, and defines the element's relative ordering for sequential focus navigation.
```

---

My original spelling: `elements'` relied on the fact that elements was plural (hence adding a trailing apostrophe). If we keep the first part of the sentence plural, but then switch to singular for the second part, it might be harder to parse.

In this suggestion, I'm making everything singular:

```suggestion
description: The `tabindex` HTML attribute make an element focusable, and defines the element's relative ordering for sequential focus navigation.
```

---

Oh, I took it to be singular even though a plural "elements" preceded it. I think your new suggestion work, let's go with that.